utils/src/parallel_restore.c

```diff
@@ -1621,6 +1622,15 @@ spr_err_t scoutfs_parallel_restore_init_slices(struct scoutfs_parallel_restore_w
 					       int nr)
 {
 	u64 total = le64_to_cpu(wri->super.total_meta_blocks);
+	/*
+	 * We have to reserve some space (10%) so there are enough free blocks after
+	 * mounting for log_merge to complete
+	 * XXX could possibly take in an arg determining percentage of space to reserve
```

Comment:
Can just drop the XXX comment here, there's no reason to call out the universal truth that we could always extend behaviour.  I'd also move the rest of the comment up to a define that contains the reservation percentage which is then used in place of the two raw instances of 10.  And 10 is probably too much, maybe 5?